src/test/java/org/kie/processmigration/service/PlanServiceImplTest.java

```diff
@@ -16,52 +16,106 @@
-import org.kie.processmigration.service.impl.PlanServiceImpl;
+import org.kie.processmigration.model.exceptions.PlanNotFoundException;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.util.List;
 
-public class PlanServiceImplTest extends AbstractPersistenceTest {
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-    @Rule
-    public WeldInitiator weld = WeldInitiator
-        .from(PlanServiceImpl.class)
-        .setPersistenceContextFactory(getPCFactory())
-        .inject(this)
-        .build();
+@QuarkusTest
+class PlanServiceImplTest {
 
     @Inject
-    private PlanService planService;
+    PlanService planService;
+
+    @AfterEach
+    @Transactional
+    void cleanUp() {
+        Plan.deleteAll();
+    }
 
     @Test
-    public void testSaveAndFindAll() {
-        // Given
-        assertNotNull(planService);
+    void testService() {
+        assertThat(planService, CoreMatchers.notNullValue());
+        assertThat(planService.findAll(), empty());
+    }
 
-        Plan plan = new Plan();
-        plan.setName("name");
-        plan.setSource(new ProcessRef().setContainerId("containerId").setProcessId("sourceProcessId"));
-        plan.setTarget(new ProcessRef().setContainerId("targetContainerId").setProcessId("targetProcessId"));
-        plan.setDescription("description");
+    @Test
+    void testCreateAndFindAll() {
+        // Given
+        Plan plan = new Plan()
+                .setName("name")
+                .setSource(new ProcessRef()
+                        .setContainerId("containerId").setProcessId("sourceProcessId"))
+                .setTarget(new ProcessRef().setContainerId("targetContainerId").setProcessId("targetProcessId"))
+                .setDescription("description");
 
         // When
-        getEntityManager().getTransaction().begin();
-        planService.create(plan);
-        getEntityManager().getTransaction().commit();
+        Plan result = planService.create(plan);
 
         // Then
         List<Plan> plans = planService.findAll();
 
-        assertNotNull(plans);
-        assertEquals(1, plans.size());
+        assertThat(plans, notNullValue());
+        assertThat(plans, hasSize(1));
+        assertThat(plans.get(0), equalTo(result));
+    }
+
+    @Test
+    void testDelete() throws PlanNotFoundException {
+        assertThrows(PlanNotFoundException.class, () -> planService.delete(1L));
+        // Given
+        Plan plan = new Plan()
+                .setName("name")
+                .setSource(new ProcessRef()
+                        .setContainerId("containerId").setProcessId("sourceProcessId"))
+                .setTarget(new ProcessRef().setContainerId("targetContainerId").setProcessId("targetProcessId"))
```

Comment:
same as before, just formatting the code
```suggestion
                .setSource(new ProcessRef().setContainerId("containerId").setProcessId("sourceProcessId"))
                .setTarget(new ProcessRef().setContainerId("targetContainerId").setProcessId("targetProcessId"))
```

Reply:
@ruromero same regarding formatting